.github/labeler.yml

```diff
@@ -39,15 +44,20 @@ area-mvc:
   - src/Mvc/**/*
   - src/MusicStore/**/*
   
-#area-platform:
+area-platform:
+  - src/Framework/**/*
 
 area-security:
   - src/Security/**/*
+  - src/Azure/AzureAD/**/*
 
+# For now, "Http" is under servers. We might adjust that later.
 area-servers:
   - src/Servers/**/*
```

Comment:
We could try applying the servers-* label for specific sub areas?

Reply:
I don't really use those anymore. I don't think they're that valuable.